ast/parser/Earthfile

```diff
@@ -8,16 +8,15 @@ RUN apk add --update --no-cache \
 
 # Install antlr.
 WORKDIR /usr/local/lib
-ARG ANTLR_VERSION=4.9.1
```

Comment:
Is this change intentional? now if we ever want to change the version we need to update it in two locations instead of just this one place. Alternatively maybe we can have curl output to just `antlr-complete.jar` so we only need to hardcode the version in one place.

Reply:
This was part of the revert. Anyway, fixed now.